distributed/deploy/spec.py

```diff
@@ -99,6 +99,44 @@ async def finished(self):
     def __repr__(self):
         return f"<{type(self).__name__}: status={self.status}>"
 
+    def _repr_html_(self):
+        self.address = getattr(self, "address", None)
+        self.external_address = None
+        self.lock = asyncio.Lock()
+        self.status = Status.created
+        self._event_finished = asyncio.Event()
+
+        attr = {
+            "address": self.address,
+            "external_address": self.external_address,
+            "lock": self.lock,
+            "status": self.status,
+            "_event_finished": self._event_finished,
```

Comment:
Same comment about `self._event_finished`. It's an internal implementation detail.